routers/api/packages/api.go

```diff
@@ -28,6 +28,11 @@ import (
 
 func reqPackageAccess(accessMode perm.AccessMode) func(ctx *context.Context) {
 	return func(ctx *context.Context) {
+		if !setting.Packages.Enabled {
```

Comment:
This gets never called because the routes are still disabled in `init.go`:

```go
	if setting.Packages.Enabled {
		r.Mount("/api/packages", packages_router.Routes())
		r.Mount("/v2", packages_router.ContainerRoutes())
	}
```

Reply:
reverted the change.